test/property.test.js

```diff
@@ -62,37 +58,127 @@ describe('loopback:property generator', function() {
     });
   });
 
-  it('creates a typed array', function(done) {
```

Comment:
Please preserve the test case for typed array and add more tests to verify "array of numbers" at least.